scripts/report_scripts/variantreport_p_sample.Rmd

```diff
@@ -1,45 +1,31 @@
 ---
 title: "SARS-CoV-2 variant report"
 date: '`r format(as.POSIXct(if ("" != Sys.getenv("SOURCE_DATE_EPOCH")) { as.numeric(Sys.getenv("SOURCE_DATE_EPOCH")) } else { Sys.time() }, origin="1970-01-01"), "%Y-%m-%d %H:%M:%S")`'
-output:
-  html_document:
-    theme: lumen
-    highlight: monochrome
-    code_folding: hide
-    depth: '3'
-    toc: yes
-    toc_float: yes
-    number_sections: yes
 params:
-  sample_name: ''
-  sigmut_db: ''
-  variants_dir: ''
-  vep_file: ''
-  snv_file: ''
-  sample_sheet: ''
+    @PARAMS_SAMPLE_NAME@
```

Comment:
This looks very wrong.

This is backwards and undoing what we've done on `main`.